packages/http/test/test-host.ts

```diff
@@ -81,6 +81,31 @@ export async function compileOperations(
   return [details, diagnostics];
 }
 
+export interface CompileOperationsResult {
+  runner: BasicTestRunner;
+  operations: HttpOperation[];
+  diagnostics: readonly Diagnostic[];
+}
+
+export async function compileOperationsFull(
+  code: string,
+  routeOptions?: RouteResolutionOptions,
+): Promise<CompileOperationsResult> {
+  const runner = await createHttpTestRunner();
+  const diagnostics = createDiagnosticCollector();
+  diagnostics.pipe(
+    await runner.compileAndDiagnose(
+      `@service(#{title: "Test Service"}) namespace TestService;
+    ${code}`,
+      {
+        noEmit: true,
```

Comment:
do you need noEmit

Reply:
It's just what the other harness that I copied from did.

Reply:
Resolving unless there's a specific reason not to include `noEmit` here.